src/main/java/com/vestmark/bitbucket/plugin/MergeConflictDetector.java

```diff
@@ -128,6 +130,19 @@ public void addResult(Branch toBranch, List<GitMergeConflict> mergeConflicts,
     mergeResults.add(new MergeResult(toBranch, mergeConflicts, messages, files));
   }
 
+  public boolean isRelated(Branch toBranch)
+  {
+    String family = toBranchName.replaceAll("release\\/([^\\.|-]*).*", "$1");
+    String toBranchFamily = toBranch.getDisplayId().replaceAll("release\\/([^\\.|-]*).*", "$1");
+    if (NumberUtils.isNumber(family) && NumberUtils.isNumber(toBranchFamily)) {
```

Comment:
Where is toBranchName coming from?

This assumes a release/ prefix, which while the default, is not necessary.
I'm fairly certain you can get the actual prefix from the container- I'll see if I can dig that answer up.
Additionally, unless you're handling it outside of this process, the default branch (master) will need to be the penultimate destination - and this logic will filter that out.

Reply:
"toBranchName" comes from pullRequest.getToRef().getDisplayId() in com.atlassian.bitbucket.pull.PullRequest.  It is set in the MergeConflictDetector constructor and is the string name of the PR destination branch.

Only branch names with a "release/" prefix will make it to this comparison, because all others are filtered out in MergeConflictDetectorServlet.java during the RepositoryBranchesRequest invocation.  So master never gets passed to this method, ever.  It is handled special in the second to last line of the doGet() method in MergeConflictDetectorServlet.java, outside of the iteration through the RepositoryBranchesRequest results.  So master will not be filtered out.

Reply:
Then the logic is broken upstream, this should handle the branch type of release, regardless of prefix.